src/kssh/ssh.go

```diff
@@ -14,3 +17,55 @@ func AddKeyToSSHAgent(keyPath string) error {
 	}
 	return nil
 }
+
+var AlternateSSHConfigFile = shared.ExpandPathWithTilde("~/.ssh/kssh-config")
+
+func CreateDefaultUserConfigFile() (bool, error) {
```

Comment:
am i reading this correctly?

this function:
1. checks ~/.ssh/kssh.config for an ssh username
2. if it's there, writes it into ~/.ssh/kssh-config
3. returns true if it did that, and false in any other case. 

my first thought is that these two configs should have names that are more different from one another. maybe the first one could be `~/.kssh/config.json` or `~/.ssh/kssh.json`. just something that distinguishes these two a little better. 
my second thought is that this function is doing a weird combination of things. maybe it should be two functions: one that ensures local ssh config reflects any changes in local kssh config (e.g. `PushKsshConfigToSsh` or `UpdateLocalSshConfig` or ...?) and another that fetches whatever the current state is of our configured situation. 

Or maybe `GetSshUserFromConfig` that returns the username from the config(s), keeps them in sync, etc. 
This seems a little more decoupled / extensible than called `Create` and returning a boolean.

Reply:
Yeah, I think you're right that having it return a boolean was a little messy. Changed to have it only return an error. What do you think of this change?